UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.Common/HealthCheck/EventHubLoggingHealthCheck.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Logging;
+using System.Threading;
+using System.Threading.Tasks;
+using UKHO.ExternalNotificationService.Common.Logging;
+
+namespace UKHO.ExternalNotificationService.Common.HealthCheck
+{
+    public class EventHubLoggingHealthCheck : IHealthCheck
+    {
+        private readonly IEventHubLoggingHealthClient _eventHubLoggingHealthClient;
+        private readonly ILogger<EventHubLoggingHealthCheck> _logger;
+
+        public EventHubLoggingHealthCheck(IEventHubLoggingHealthClient eventHubLoggingHealthClient, ILogger<EventHubLoggingHealthCheck> logger)
+        {
+            this._eventHubLoggingHealthClient = eventHubLoggingHealthClient;
+            this._logger = logger;
+        }
+
+        public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, CancellationToken cancellationToken = default)
+        {
+            var healthCheckResult = await _eventHubLoggingHealthClient.CheckHealthAsync(context);
```

Comment:
Please pass `cancellationToken`:
```suggestion
            var healthCheckResult = await _eventHubLoggingHealthClient.CheckHealthAsync(context, cancellationToken);
```

Reply:
Yes, I have passed cancellationToken as parameter